tests/cypress/e2e/jcontent/compareStagingLive.cy.ts

```diff
@@ -0,0 +1,72 @@
+import {JContent} from '../../page-object';
+import gql from 'graphql-tag';
+
+describe('JContent preview tests', () => {
+    const addContent = gql`mutation MyMutation {
+      jcr {
+        addNode(
+          name: "addedNode"
+          parentPathOrId: "/sites/jcontentSite/home/area-main"
+          primaryNodeType: "jnt:bigText"
+          properties: { name: "text", language: "en", value: "test added" }
+        ) {
+          uuid
+        }
+      }
+    }`;
+
+    const path = 'pages/home#(jcontent:(compareDialog:(open:!t,path:/sites/jcontentSite/home)))';
+
+    beforeEach(() => {
+        cy.executeGroovy('jcontent/createSite.groovy', {SITEKEY: 'jcontentSite'});
+        cy.apollo({mutationFile: 'jcontent/createContent.graphql'});
+        cy.loginAndStoreSession(); // Edit in chief
+        const jcontent = JContent.visit('jcontentSite', 'en', 'pages/home');
+        jcontent.publishAll();
+        cy.get('button[data-sel-role="openInLive"]', {timeout: 5000}).should('be.visible');
```

Comment:
maybe possible to replace with publishAndWaitJobEnding method from jahia-cypress: https://github.com/Jahia/jahia-cypress/blob/main/src/utils/PublicationAndWorkflowHelper.ts#L2 as going through UI to publish adds quite a bit of time per test.